e2e/runner_test.go

```diff
@@ -66,6 +66,12 @@ var supportedProducts = map[string]*contextMetadata{
 		},
 		modules: []string{"metricbeat"},
 	},
+	"parity-tests": &contextMetadata{
```

Comment:
For newcomers, long story about this map here: https://github.com/elastic/metricbeat-tests-poc/pull/104